osquery/config/plugins/tls.cpp

```diff
@@ -67,16 +73,23 @@ Status TLSConfigPlugin::setUp() {
   }
 
   uri_ = TLSRequestHelper::makeURI(FLAGS_config_tls_endpoint);
-
-  // If the initial configuration includes a non-0 refresh, start an additional
-  // service that sleeps and periodically regenerates the configuration.
-  if (FLAGS_config_tls_refresh >= 1) {
-    Dispatcher::addService(std::make_shared<TLSConfigRefreshRunner>());
-  }
+  started_thread = false;
```

Comment:
You can drop this and initialize to `false` using `{false}` after the name in the header/class definition.